client/src/components/MapMarker/styles.js

```diff
@@ -0,0 +1,40 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'column',
+    alignSelf: 'flex-start',
+  },
+  bubble: {
+    flex: 0,
+    flexDirection: 'row',
+    alignSelf: 'flex-start',
+    backgroundColor: '#5A6DFF',
+    padding: 2,
+    borderRadius: 3,
+    borderColor: '#161AF9',
+    borderWidth: 0.5,
+  },
+  name: {
+    color: '#FFFFFF',
+    fontSize: 13,
+  },
+  arrow: {
+    backgroundColor: 'transparent',
+    borderWidth: 4,
+    borderColor: 'transparent',
+    borderTopColor: '#FF5A5F',
```

Comment:
You can avoid hardcoding colors by referencing:

`import Colors from '../../themes/Colors';`


 `backgroundColor: Colors.bgBtn,`

Reply:
in future we will need a few different colors for this, ill hold off on linking back to theme colors until i link that code in

Reply:
ok - sounds great - typically we would have a few set theme colors we re-use - then we can tweak in one place if changes are required.